en/firefox/browsers/compare/opera.ftl

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/compare/opera/
+
+# HTML page title
+compare-opera-firefox-vs-opera-which-is = { -brand-name-firefox } vs. { -brand-name-opera }: Which is the better browser for you?
+
+# HTML page description
+compare-opera-compare-opera-to-the-firefox = Compare { -brand-name-opera } to the { -brand-name-firefox-browser } to find out which is the best browser for you.
+
+# Main title
+compare-opera-comparing-firefox-browser = Comparing { -brand-name-firefox-browser } with { -brand-name-opera }
+
+compare-opera-operas-privacy-policy-lacks = Opera’s privacy policy lacks some specificity in its explanation of which types of information it collects and how, in certain sections, it says they collect names of account holders, IP addresses and search terms. What seems confusing and troubling is the section about International data transfers; when, how often and why they need to transfer your data internationally is not explained.
```

Comment:
1). `Opera`
2). Change comma to a period followed by a capital I so this long sentence breaks into two: `... it collects and how. In certain sections...`